src/cli/self_update.rs

```diff
@@ -159,8 +159,8 @@ The Cargo home directory located at:
 
 This can be modified with the CARGO_HOME environment variable.
 
-The `cargo`, `rustc`, `rustup` and other commands will be added to
-Cargo's bin directory, located at:
+The `cargo`, `rustc` and other commands will be added to Cargo's
+bin directory, located at:
```

Comment:
This change claims `rustup` will no longer be put into `$CARGO_HOME/bin` but your change still puts it there, via symlink.

Reply:
I have undone this.